lib/diagnostic.md

```diff
@@ -54,7 +70,8 @@ Please write a method that traverses from the lister DOM node (one of the list
 item elements) to the body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+var body = $(lister).parent().parent();
+body.hide();
```

Comment:
Good, but it's okay and common practice to string these together. also your selector needs to be inside quotes.

$('lister').parent().parent().hide()